controls/raddiagram/features/raddiagrams-serialization.md

```diff
@@ -313,10 +314,50 @@ For every other property that you need to be part of the Serialization/Deseriali
 	End Sub	
 {{endregion}}
 
->Please note that the __Double.ToInvariant()__ extension method is defined in the __Telerik.Windows.Diagrams.Core__ namespace and in order to use it in your application, you'll have to add a using statement: using Telerik.Windows.Diagrams.Core;
+>Please note that the `Double.ToInvariant()` extension method is defined in the __Telerik.Windows.Diagrams.Core__ namespace and in order to use it in your application, you'll have to add a using statement: using Telerik.Windows.Diagrams.Core;
 
 >tip If you decide to manually save one of the [properties](#extending-raddiagram-serialization) automatically saved by the diagram, make sure to give it a custom key (different from the default one used for saving). Also, implement both, the saving and loading logic in the ShapeSerialize and ShapeDeserialized event handlers. For example, if you want to manually save the FontWeight property of a RadDiagramShape, you can use a key different than "FontWeight" in the SerializationInfo (example: "MyFontWeight"). We recommend this as a precaution, because internally the diagram parses the value given in the SerializationInfo and if it doesn't match a specific format an exception will be thrown. 
 
+## Metadata Manual Serialization/Deserialization

```

Comment:
Use a more descriptive title. This MetadataSerialization API is very strange. As I understand the event is fired to save/load properties of the RadDiagram instance. THe diagram itself doesn't have anything related to "metadata" in its API, except this event. In other words, it is not very clear what is does and what "metadata" means.

IMO, the title of this section should be something like "Save/Load RadDiagram Properties".